### ISSUE TITLE
PH2031 False positive

### ISSUE THREAD

**Issue by ynsehoornenborg** [philips-software/roslyn-analyzers#703]
If raises a Diagnostic on:
``` cs
double.TryParse(timeParts[0], NumberStyles.Any, CultureInfo.InvariantCulture, out var time))
```

**Comment by ynsehoornenborg** [comment]
My mistake
